api/org/commcare/api/persistence/SqlSandboxUtils.java

```diff
@@ -13,13 +13,25 @@
  * @author wspride
  */
 public class SqlSandboxUtils {
+
+    /**
+     * Used by touchforms
+     */
+    public static UserSqlSandbox getStaticStorage(String username, String path) {
+        return new UserSqlSandbox(username, path);
+    }
+
     public static UserSqlSandbox getStaticStorage(String username) {
-        return new UserSqlSandbox(username);
+        return new UserSqlSandbox(username, null);
```

Comment:
There's a UserSqlSandbox constructor that doens't require an argument for the second field, we should use that instead of assuming what the default should be for clarity